src/coord/src/coord.rs

```diff
@@ -2217,18 +2217,18 @@ impl Coordinator {
         object_columns: usize,
         desc: RelationDesc,
     ) -> Result<ExecuteResponse, CoordError> {
+        // TAIL can be in a transaction if it's the only operation. An exception
```

Comment:
nit: this comment + the surrounding branch was confusing to me because of multiple levels of negatives

i.e. the comment says "we will make an exception if the tail has a timestamp"
the code says "if the tail does not have a timestamp -- do add_transaction_ops", and it took me a while to understand that the "exception" was "we won't call add_transaction_ops" -- not sure if there's anything super actionable here but maybe consider rewording?

Reply:
Done.